src/components/Settings/StampPaletteTab/StampPaletteEditorBasicInfo.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div :class="$style.container">
+    <label :class="$style.formGroup">
+      <p :class="$style.label">パレット名</p>
```

Comment:
```suggestion
      <label :class="$style.label">パレット名</label>
```

かな 元のは div で囲っちゃって OK です

Reply:
なるほど　そしたらこのままで良さそうです